controller_manager:
  ros__parameters:
    update_rate: 10

    position_controller:
      type: position_controllers/JointGroupPositionController

position_controller:
  ros__parameters:
    joints:
      - joint_1

    command_interfaces:
      - position

    state_interfaces:
      - position
